dk-installer.py

```diff
@@ -125,7 +125,15 @@ def generate_password():
 
 def write_credentials_file(product, lines):
     try:
-        file_name = CREDENTIALS_FILE.format(product)
+        if detect_os() == 'Windows':
+            home_dir = pathlib.Path.home()
+            credentials_folder = home_dir.joinpath("Documents", "dk-installer")
+            if not credentials_folder.exists():
+                credentials_folder.mkdir(parents=True, exist_ok=True)
+            os.makedirs(credentials_folder, exist_ok=True)
+            file_name = os.path.join(credentials_folder, CREDENTIALS_FILE.format(product))
+        else:
+            file_name = CREDENTIALS_FILE.format(product)
```

Comment:
You don't need to check the folder existence, given you are calling `mkdir` with `exist_ok=True`. It's also doing it multiple times. I'm also suggesting some readability/DRY changes.

```suggestion
        if detect_os() == 'Windows':
            credentials_folder = pathlib.Path.home() / "Documents" / "dk-installer"
            credentials_folder.mkdir(parents=True, exist_ok=True)
        else:
            credentials_folder = pathlib.Path()
        file_name =  credentials_folder / CREDENTIALS_FILE.format(product)
```